tools/start_server.py

```diff
@@ -249,11 +250,11 @@ def start_app_file_server(self, _app_storage_folder):
     secret_key = str(uuid.uuid4())
     print('\033[2mPort is %s, secret key is %s\033[0m' % (port, secret_key))
 
-    process = subprocess.Popen(['python3', '../../tools/file_server.py', str(port), secret_key], cwd=_app_storage_folder)
+    process = subprocess.Popen(['python3', '../../tools/file_server.py', str(self.WEBENGINE_HOST), str(port), secret_key], cwd=_app_storage_folder)
     time.sleep(1)
 
     WebEngineManager.next_available_port += 1
-    return {'process': process, 'url': 'http://localhost:%s/%s/' % (port, secret_key)}
+    return {'process': process, 'url': 'http://%s:%s/%s/' % (self.WEBENGINE_HOST, port, secret_key)}
```

Comment:
`self.WEBENGINE_HOST` will be the client's IP, not the file server IP. This will only work on localhost.

I think this value should just be user defined at the top of the file or as an argument and default to 'localhost'

Reply:
Added hostname config option in https://github.com/smartdevicelink/generic_hmi/pull/272/commits/6ef0b3ca6d19109c44247b0d88aab322f642f98c